.github/workflows/reusable-release.yml

```diff
@@ -0,0 +1,248 @@
+name: Publish npm packages (reusable)
+
+on:
+  workflow_call:
+    inputs:
+      release_type:
+        required: true
+        type: string
+      branch:
+        required: false
+        type: string
+    secrets:
+      gh_token:
+        required: true
+      npm_token:
+        required: true
+
+env:
+  # Dagger
+  DAGGER_PLAN: cue.mod/pkg/github.com/3box/pipeline-tools/ci/plans/ceramic.cue
+  DAGGER_LOG_FORMAT: "plain"
+  PIPELINE_TOOLS_VER: "0.2.0"
```

Comment:
Just wondering, is there a way to always pull the latest release in pipeline-tools instead? or is that bad practice I guess?

Reply:
Yeah, we could, I just get worried we'd pull in some breaking change and all the pipelines will break across Ceramic/CAS/IPFS, etc.

You know what, it's probably ok. The change I'm making right now in pipeline-tools will let us test workflows in feature branches so we don't have to worry about breaking other repos by pushing untested changes to develop.